tests/test_sklearn_random_forest_converters.py

```diff
@@ -187,6 +196,41 @@ def test_model_multi_class_nocl(self):
                           "StrictVersion(onnxruntime.__version__)"
                           " <= StrictVersion('0.2.1')")
 
+    def common_test_model_hgb_regressor(self, add_nan=False):
```

Comment:
Where is the unit test for HistGradientBoostingClassifier converter?

Reply:
Below.

Reply:
Not done yet in fact. In a different PR a in this one if this seems already ok.